publishexe.targets

```diff
@@ -38,6 +37,11 @@
   <Target Name="FixUpCoreConsoleNames">
     <Move SourceFiles="$(Outdir)$(AssemblyName).exe" DestinationFiles="$(Outdir)$(AssemblyName).dll" />
     <Move SourceFiles="$(Outdir)CoreConsole.exe" DestinationFiles="$(Outdir)$(AssemblyName).exe" />
+    <!-- create a marker indicating that the file was renamed, this will be used during signing to sign the correct file -->
+    <Touch Files="$(Outdir)$(AssemblyName).publishedexe" AlwaysCreate="true" />
+    <ItemGroup>
```

Comment:
The marker file is renamed in GetFilesToSignItems in sign.builds (e.g. GenAPI.exe -> GetAPI.dll).  We cannot rename it in FixUpCoreConsoleNames as the marker hasn't been written yet.  We could do the renaming here (avoiding this other marker) however we would have to ensure that this target is always executed after the marker file has been written.